README.rst

```diff
@@ -21,6 +21,22 @@
 
 Get coala to lint all your languages in your project with one tool and config!
 
+Introduction
+------------
+
+coala is a language independent analysis toolkit. It empowers developers
```

Comment:
in my opinion we should bold out coala everywhere in the docs, because it's the freakin' name :p we should point it out more